autofetch/views.py

```diff
@@ -0,0 +1,93 @@
+import requests
+from bs4 import BeautifulSoup
+from django.shortcuts import render
+from external.craigslist import fetch_craigslist_housing
+from external.googleapi.fetch import fetch_reverse_geocode
+from location.models import Location
+from location.models import Apartment
+from datetime import datetime
+
+def autofetch(request):
+
+    print(f"Start: {str(datetime.now())} \n")
+
+    city_list = ["brx", "brk", "fct", "lgi", "mnh", "jsy", "que", "stn", "wch"]
```

Comment:
In `autofetch` function, variables are the following:
- city_list
- limit

It would be good to have those variables as GET parameters. For instance,
```
http://master-branch.herokuapp.com/autofetch?limit=100&city=brk,mnh
```

Reply:
Good point! 
1. So do you think should we keep the variable limit?
2. Does having the city as variable intend to let us specify the city we want to update? And yes, maybe by doing this we can split the huge request into several small requests.

Reply:
1. Yes
2. Yes

By doing this, we don't need to re-run or re-deploy our django application in order to change our crawling strategy.

Reply:
Will do this. Thanks!

Reply:
Updated.
51c645b